lib/internal/fs/rimraf.js

```diff
@@ -1,9 +1,4 @@
-// This file is a modified version of the rimraf module on npm. It has been
-// modified in the following ways:
-// - Use of the assert module has been replaced with core's error system.
-// - All code related to the glob dependency has been removed.
-// - Bring your own custom fs module is not currently supported.
-// - Some basic code cleanup.
+// This file is a modified version of the rimraf module on npm.
```

Comment:
I think it was added to address https://github.com/nodejs/node/pull/29168#discussion_r315815264. Since we can see the changes made to the file with a `git log` and the linked PRs have the reasons for the changes, it feels redundant to keep this around.